ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/model/CaseData.java

```diff
@@ -220,7 +220,7 @@
     @CCD(
         label = "How did the petitioner change their name?"
     )
-    private Set<ChangedNameHow> d8PetitionerNameChangedHow;
+    private ChangedNameHow d8PetitionerNameChangedHow;
```

Comment:
If you change field type then elastic search will be broken.
Think you need to bump the case type  too with this PR https://github.com/hmcts/nfdiv-case-api/blob/master/ccd-definitions/src/main/java/uk/gov/hmcts/reform/divorce/ccd/ccdcase/NoFaultDivorce.java#L22 to `NO_FAULT_DIVORCE4`.
Other option is to ask platops to delete ES indexes and then this should be fine.